brokerapi/brokers/gcp_service_broker.go

```diff
@@ -589,6 +598,13 @@ func InitCatalogFromEnv() ([]models.Service, error) {
 	}
 	servicePlans[bigtableServiceId] = append(servicePlans[bigtableServiceId], bigtablePlans...)
 
+	// set up bigtable custom plans
+	spannerPlans, spannerServiceId, err := getDynamicPlans("SPANNER_CUSTOM_PLANS", spanner.MapPlan)
```

Comment:
did you consider extracting this custom plan save out to a helper method? It seems like it's general enough between the 3 but I could be wrong there. It's kinda the next step after your `getDynamicPlans` refactor.

I'd weigh if it was worth it here or not. Might be too dense to be worth it.